File: java/src/com/f12developers/encapsulation/Accounts.java
package com.f12developers.encapsulation;

public class Accounts {
    // Field
    private float balance;


    // methods
    public void deposit(float amount){
        if(amount > 0)
            balance += amount;
    }
    public void withdraw(float amount){
        if (amount > 0)
            balance -= amount;
    }

    public float getBalance() {
        return balance;
    }
}

File: Dart/encapsulation/Accounts.dart
import 'dart:wasm';

class Accounts {
  
  double _balance = 0;

  void deposit(double amt) {
    if (amt > 0) {
      _balance += amt;
    }
  }

  void withdraw(double amt) {
    if (amt > 0) {
      _balance -= amt;
    }
  }

  double getBalance(){
    return _balance;
  }
}
